merlin/models/tf/transforms/bias.py

```diff
@@ -243,3 +282,21 @@ def _check_items_cardinality(self, item_freq_probs):
                 f"(expected {cardinalities[item_id_feature_name]}"
                 f", got {tf.shape(item_freq_probs)[0]})"
             )
+
+    def compute_call_output_shape(self, input_shapes):
+        return self.compute_output_shape(input_shapes)
+
+    def get_config(self):
```

Comment:
> After adding the methods to the class, a new issue was raised:
> 
> ==>` If loading the class as a standalone using get_config/from_config methods, the tf Variable candidate_probs defined [here](https://github.com/NVIDIA-Merlin/models/blob/8f1a73a07fc4586a374d9d7c793111946726517b/merlin/models/tf/transforms/bias.py#L129) is not loaded.
==> Loading a whole model containing logQ block using tf.keras.load() is correctly loading the variable candidate_probs .

Is the above issue resolved? And if so, what changed here to fix this?

Reply:
I think maybe the issue is that we also need to save some more of the params in this `get_config` - the `item_freq_probs` and `candidate_tag_id`